embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/comms/api/ApiResponseCache.kt

```diff
@@ -76,7 +79,7 @@ class ApiResponseCache(
             val uri = URI.create(url)
             val requestMethod = request.httpMethod.toString()
             val headerFields = request.getHeaders().mapValues { listOf(it.value) }
-            obj.get(uri, requestMethod, headerFields)
+            obj.get(uri, requestMethod, headerFields.plus("Accept-Encoding" to listOf("gzip")))
```

Comment:
`request` has an `acceptEncoding` property that will set this header, IMO it'd be preferable to set the value in `EmbraceApiService#prepareConfigRequest` rather than adding a special case here

Reply:
Yup - that's what I did. You can just pass it in the constructor and it'll work fine.